README.md

```diff
@@ -1,70 +1,33 @@
-# Getting Started with Create React App
+# React Calculator - By HeflerDev
```

Comment:
### [Optional]
Do well to add more information to your readme file, take a look at the [Microverse template](https://github.com/microverseinc/readme-template) from pathwright